### ISSUE TITLE
Specify StatMech modes using Excel

### ISSUE THREAD

**Issue by jonlym** [VlachosGroup/pMuTT#48]
Currently, StatMech models can only be initialized using the "statmech_model" header and the `presets` dictionary. The user should be able to initialize modes individually so they are not limited to the presets.

**Comment by jonlym** [comment]
The following rows are now considered equivalent when importing from Excel. I believe if ``statmech_model`` is before the models specific to certain modes then the preset can be modified but further testing is required.

| name          | elements.N | elements.H | phase | statmech_model | trans_model | vib_model   | rot_model  | elec_model | symmetrynumber | atoms        | potentialenergy | vib_wavenumber |
|---------------|------------|------------|-------|----------------|-------------|-------------|------------|------------|----------------|--------------|-----------------|----------------|
| N2            | 2          | 0          | G     | IdealGas       |             |             |            |            | 2              | ./N2/CONTCAR | -16.63          | 2744.00        |
| N2_individual | 2          | 0          | G     |                | IdealTrans  | HarmonicVib | RigidRotor | IdealElec  | 2              | ./N2/CONTCAR | -16.63          | 2744.00        |

**Comment by wittregr** [comment]
Need additional functionality to allow a user to specify a preset statmech_model and then override one of the presets by specifying a different trans, vib, rot, elec, etc_model. The order of the statmech_model preset and any overriding models should not be required.

**Comment by jonlym** [comment]
The `read_excel` function was updated so `statmech_model` columns do not overwrite previously set modes.

The code was tested using the following code and spreadsheet:
```python
import os
from pprint import pprint
from pmutt.io.excel import read_excel

os.chdir(os.path.dirname(__file__))

all_species = read_excel(io='test_sheet.xlsx')

print('Name\tVibrational Class Assigned\tExpected Vibrational Class')
for ind_species in all_species:
    print('{}\t{}\t{}'.format(ind_species['name'],
                              ind_species['vib_model'],
                              ind_species['notes']))
```
[test_sheet.xlsx](https://github.com/VlachosGroup/pMuTT/files/3576444/test_sheet.xlsx)

We tested several scenarios:
1. Only `statmech_model`
2. `vib_model` specified before `statmech_model`
3. `vib_model` specified after `statmech_model`
4. `EmptyMode` assigned to `vib_model` (this functionality was not possible beforehand).

The results are below:

| Name | Vibrational Class Assigned               | Expected Vibrational Class             |
|------|------------------------------------------|----------------------------------------|
| H2_1 | <class 'pmutt.statmech.vib.HarmonicVib'> | Vibrational mode should be HarmonicVib |
| H2_2 | <class 'pmutt.statmech.vib.QRRHOVib'>    | Vibrational mode should be QRRHOVib    |
| H2_3 | <class 'pmutt.statmech.vib.QRRHOVib'>    | Vibrational mode should be QRRHOVib    |
| H2_4 | <class 'pmutt.statmech.EmptyMode'>       | Vibrational mode should be EmptyMode   |

This fix will be in pMuTT version 1.2.13
